org.eclipse.debug.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.debug.core; singleton:=true
-Bundle-Version: 3.19.100.qualifier
+Bundle-Version: 3.20.0.qualifier
```

Comment:
Changes are to internal files, Why is minor version increased?

Reply:
So, the resources bundle (updated to use the getLineSeparator API) is being re-exported from this one. Therefore since that has a new minor version, this one needs to have a minor version bump as well. See [this build failure](https://ci.eclipse.org/platform/job/eclipse.platform.debug/job/PR-28/7/console):

```
* Marker [on: /org.eclipse.debug.core/META-INF/MANIFEST.MF, id: 11, type: org.eclipse.pde.api.tools.version_numbering, attributes: [apiMarkerID: 4, charEnd: 168, charStart: 150, description: , lineNumber: 5, message: The minor version should be incremented in version 3.19.200, because the modification of the version range for the re-exported bundle org.eclipse.core.resources requires a minor version change, messagearguments: 3.19.200#org.eclipse.core.resources, problemid: 929038356, severity: 2, sourceId: API Tools, version: 3.20.0], created: 6/13/22, 8:18 PM]
```

Reply:
I think this is ok to resolve?